exporter/sumologicexporter/exporter.go

```diff
@@ -456,3 +456,25 @@ func getSignalURL(oCfg *Config, endpointURL string, signal pipeline.Signal) (str
 
 	return url.String(), nil
 }
+
+func sanitizeURL(urlString string) string {
+	leftIndex := strings.Index(urlString, "otlp/")
+	rightIndex := strings.LastIndex(urlString, "/v1/")
+	if leftIndex == -1 || rightIndex == -1 {
+		return urlString
+	}
+	s1 := urlString[0 : leftIndex+5]
```

Comment:
Nit: Can we put `otlp/` in a string variable, then have another int variable that represents its length? This would make the later indexing logic with +5 and -5 a bit more clear.